tools/packaging/packaging.psm1

```diff
@@ -3090,7 +3090,8 @@ function New-DotnetSdkContainerFxdPackage {
     [CmdletBinding(SupportsShouldProcess = $true)]
     param(
         [Parameter(Mandatory)] $FxdPackagePath,
-        [Parameter(Mandatory)] $ReleaseTag
+        [Parameter(Mandatory)] $ReleaseTag,
```

Comment:
It's good to validate this.  You can find the regex in other places in the code.  It helps if someone messes up passing this parameter into the script.

Consider similar issues on the other parameters.

Reply:
Fixed